### PR TITLE
Changes are made to insert a new row into a specific location of grid by passing rowIndex parameter to addNewRow(ADD_NEW_ROW) action.

### PR DESCRIPTION
**Pull request by underwater222**
I have test it to add a new row into a specific location and confirmed that it works. However it fails some testing on 'npm t'. Please take a look.

### CODE REVIEW

File: src/reducers/actionHelpers/datasource.js [hunk 1]
```diff
@@ -160,7 +160,8 @@ export const addNewRow = (state, { rowId, stateKey }) => {
         data = new List();
     }
 
-    const newData = data.unshift(newRow);
+    rowIndex = rowIndex===undefined?0:rowIndex;
```
Comment (bencripps) [inline src/reducers/actionHelpers/datasource.js]:
```js
rowIndex = rowIndex === undefined
    ? 0
    : rowIndex;
```

Comment (underwater222) [inline src/reducers/actionHelpers/datasource.js]: Greate! Thanks.

Comment (bencripps) [inline src/reducers/actionHelpers/datasource.js]: No problem, once you push the change, I will merge!

**Comment by underwater222** [comment]
I have push the coding style change for "? :" operator . Do I need to create a new pull request or can you merge from this pull request on my "addnewrowatpos" branch?

**Pull request merged**